test/libsolidity/syntaxTests/scoping/external-library-function-inside-scope.sol

```diff
@@ -0,0 +1,9 @@
+library L {
```

Comment:
I looked for, but could not find a test case for this situation. Although unrelated to the PR, maybe easier than creating an issue/PR just for the sake of it (?)

Reply:
It's not completely unrelated and it's small so fine to keep it here.